.github/workflows/coverage.yml

```diff
@@ -19,7 +19,9 @@ env:
 
 jobs:
   codecov:
-    runs-on: ubuntu-latest
+    # use the oldest kernel supported by github's CI
+    # https://github.com/actions/virtual-environments
```

Comment:
Probably worth adding a reminder in these comments to update the minimum supported kernel version when changing (even if we haven't documented it yet)

Reply:
Added.